rust/openvasd/src/main.rs

```diff
@@ -24,7 +24,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     if !config.ospd.socket.exists() {
         tracing::warn!("OSPD socket {} does not exist. Some commands will not work until the socket is created!", config.ospd.socket.display());
     }
-    let scanner = scan::OSPDWrapper::new(config.ospd.socket.clone());
+    let mut scanner = scan::OSPDWrapper::new(config.ospd.socket.clone());
+    if let Some(rtimeout) = config.ospd.read_timeout {
```

Comment:
I think it would be simpler to read when this is done within the new method by just passing the option into new.